README.md

```diff
@@ -22,25 +22,27 @@ The digital twin will:
 
 ## Key Functionalities
 
-- **1** Users can access the CROP platform and database using multiple ways.
-- **2** The CROP web application is the main interface for the digital twin. Users can explore collected heterogeneous IoT sensor data, analyse farm conditions at various points in time, use the developed 3D visualisation tools.
-- **3** The CROP database is constantly updated from multiple streams of data: Hyper API, Stark energy usage platform, and others.
-- **4** CROP machine learning services integrate automated prediction and calibration models into the platform.
-- **5** The Unity 3D model is found in [this repo](http://github.com/alan-turing-institute/CROP_unity): 
+- The CROP web application is the main interface for the digital twin. Users can
+    - explore collected heterogeneous IoT sensor data,
+    - analyse farm conditions at various points in time,
+    - use the interactive 3D visualisation of the farm,
+    - forecast future farm conditions using machine learning models built into the platform.
+- The CROP database is constantly updated from multiple streams of data: Hyper API, Stark energy usage platform, and others.
+- For forecasting, we CROP uses two models
+    - An ARIMA model uses past temperature and relative humidity data in the farm to forecast conditions a few days into the future.
+    - A GES model uses past sensor data, weather data, farm operational parameters (lighting schedules, fan settings, etc.) and Gaussian processes to forecast conditions a few days into the future. The GES model has the ability to predict various alternative scenarios, such as how would conditions change if the lights were switched on at a different time, or the fan settings were changed.
```

Comment:
Maybe here I would restructure the sentence slightly, to separate the method from the inputs, e.g. "The GES model uses ... ,... to forecast conditions a few days into the future, using Gaussian Process Regression."  ?
(Also, do we know what "GES" stands for ?)